cmd/server/assets/login/login.html

```diff
@@ -22,7 +22,7 @@
               <form id="login-form" class="floating-form" action="/" method="POST">
                 <div class="form-label-group">
                   <input type="email" id="email" name="email" class="form-control" placeholder="Email address" required
-                    autofocus />
+                    autofocus {{if .currentUser}}disabled value="{{.currentUser.Email}}"{{end}}/>
```

Comment:
is there an easy way to escape off this page and log in as someone else?

Reply:
Nope. Added the nav header in.

This is currently a bit of a dead end anyway since it's not linked in anywhere